modules/cloud_account/aws/0.2/facets.yaml

```
@@ -0,0 +1,83 @@
+intent: cloud_account
+flavor: aws
+version: '0.2'
+description: Module to configure and provision resources within a cloud account
+clouds:
+- aws
+spec:
+  description: Module to configure and provision resources within a cloud account
+    (AWS, GCP, Azure)
+  properties:
+    region:
+      type: string
+      title: AWS Region
+      description: AWS region where the VPC will be created
+      x-ui-overrides-only: true
+      x-ui-api-source:
+        endpoint: /cc-ui/v1/dropdown/aws/regions-v2
+        method: GET
+        labelKey: region
+        valueKey: region
+        path: regions
+    cloud_account:
+      description: Select the cloud account to use (must be previously linked)
+      title: Cloud Account
+      type: string
+      x-ui-typeable: true
+      x-ui-overrides-only: true
+      x-ui-api-source:
+        endpoint: /cc-ui/v1/accounts/
+        method: GET
+        labelKey: name
+        valueKey: id
+        filterConditions:
+        - field: accountType
+          value: CLOUD
+        - field: provider
+          value: AWS
+        params:
+          includeContent: false
+  required:
+  - cloud_account
+  - region
+  title: Cloud Account Manager
+  type: object
+  x-ui-order:
+  - cloud_provider
+  - cloud_account
+  - region
+outputs:
+  default:
+    type: '@outputs/cloud_account'
```

change the output type
```suggestion
    type: '@facets/aws_cloud_account'
```